README.md

```diff
@@ -643,6 +644,20 @@ spec:
 
 **Note**: The `image` and `image_version` are intended for local mirroring scenarios. Please note that using a version of AWX other than the one bundled with the `awx-operator` is **not** supported. For the default values, check the [main.yml](https://github.com/ansible/awx-operator/blob/devel/roles/installer/defaults/main.yml) file.
 
+#### Container security context
+To change/add the security context for a specified container, add your items as key-value pairs under the name of the container in question. Possible container names are `redis`, `init`, `ee`, `task`, `web`, `rsyslog`. 
+
+```yaml
+---
+spec:
+  security_context_settings:
+    task:
+      runAsUser: 1000
+      runAsGroup: 1000
+    rsyslog:
+      runAsUser: 1000
+```
+
 #### Redis container capabilities
 
 Depending on your kubernetes cluster and settings you might need to grant some capabilities to the redis container so it can start. Set the `redis_capabilities` option so the capabilities are added in the deployment.
```

Comment:
I think it may make more sense to change the example here to just use `security_context_settings.redis` parameter.  That way we don't have users attempting to use both methods of setting capabilities and end up with one overriting the other.  And we could mark `redis_capabilities` as a deprecated field (but leave in the conditional in the template for backwards compat in the meantime).  What do you think?

Reply:
Thanks for your comment, I made changes to readme file, as well as Openshift form descriptors. How would you like to mark field`redis_capabilities` as depracated?  Would be sufficient just saying it readme, i.e. this field will become deprecated in future version, please use `security_context_settings` instead?